### PR TITLE
Create backstage deployment configuration, add individual Dockerfiles…

### PR DESCRIPTION
**Pull request by catalindev0ps**
… for backend and frontend KNDP Backstage applications

### CODE REVIEW

File: .github/workflows/releases.yaml [hunk 1]
```diff
@@ -48,26 +48,32 @@ jobs:
...
       - name: Prepare metadata
         id: metadata
         run: |
-          git diff --name-only HEAD~1 HEAD | awk -F'/' 'NF >= 2 { print $1 "/" $2 }' | uniq > build.txt;
+          git diff --name-only HEAD~1 HEAD | awk -F'/' 'NF >= 2 { print $1 "/" $2 "/" $3 }' | uniq > build.txt;
 
       - name: Build images
         id: build
-        run: cat build.txt | xargs -I{} bash -c '[ -f {}/Dockerfile ] && docker build -t ${REGISTRY}/${{ github.repository }}/$(basename {}):$release_version -f {}/Dockerfile . && docker push ${REGISTRY}/${{ github.repository }}/$(basename {}):$release_version || true'
+        run: cat build.txt | xargs -I{} bash -c '[ -f {}/Dockerfile ] && docker build -t ${REGISTRY}/${{ github.repository }}/$(basename {}):0.1.0 -f {}/Dockerfile . && docker push ${REGISTRY}/${{ github.repository }}/$(basename {}):0.1.0 || true'
```
Comment (web7evgheni) [inline .github/workflows/releases.yaml:76]: @catalindev0ps Why version is hardcoded?

File: packages/backstage/charts/backstage-app/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      {{- include "backstage-app.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "backstage-app.labels" . | nindent 8 }}
+	{{- with .Values.podLabels }}
```
Comment (web7evgheni) [inline packages/backstage/charts/backstage-app/templates/deployment.yaml:22]: @catalindev0ps Please fix formatting.

File: tsconfig.json [hunk 3]
```diff
@@ -1,10 +1,8 @@
 {
   "extends": "@backstage/cli/config/tsconfig.json",
   "include": [
-    "packages/*/src",
-    "plugins/*/src",
-    "plugins/*/dev",
-    "plugins/*/migrations"
+    "packages/backstage/*/src",
+    "plugins/backstage/*/src"
```
Comment (web7evgheni) [inline tsconfig.json:5]: @catalindev0ps we not have ./plugins directory

File: packages/backstage/backend/package.json [hunk 4]
```diff
@@ -32,7 +32,6 @@
     "@backstage/plugin-search-backend-module-pg": "^0.5.15",
     "@backstage/plugin-search-backend-node": "^1.2.10",
     "@backstage/plugin-techdocs-backend": "^1.8.0",
-    "app": "link:../app",
```
Comment (web7evgheni) [inline packages/backstage/backend/package.json:35]: @catalindev0ps Why you remove it? This is package of Yarn workspace

**Review by web7evgheni — CHANGES_REQUESTED** [review]

File: packages/backstage/backend/package.json (hunk 4 above)
Comment (catalindev0ps) [reply packages/backstage/backend/package.json:35]: @web7evgheni otherwise the backend will build to the frontend

Comment (web7evgheni) [reply packages/backstage/backend/package.json:35]: @catalindev0ps Not clear what you mean.

File: .github/workflows/releases.yaml (hunk 1 above)
Comment (catalindev0ps) [reply .github/workflows/releases.yaml]: Backstage backend by default is engaged to build and serve the frontend, but now i modified both to build and serve individually

File: packages/backstage/charts/backstage-app/values.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+#     optional: false
+
+# Additional volumeMounts on the output Deployment definition.
+volumeMounts: []
+# - name: foo
+#   mountPath: "/etc/foo"
+#   readOnly: true
+
+appConfig:
+  name: APP_COFIG_app_baseUrl
```
Comment (web7evgheni) [inline packages/backstage/charts/backstage-app/values.yaml:40]: @catalindev0ps Do you think just single config we will required to setup? Please make array

File: packages/backstage/charts/backstage-backend/templates/app-config-inject.yaml
Comment (web7evgheni) [inline packages/backstage/charts/backstage-backend/templates/app-config-inject.yaml:1]: @catalindev0ps After was integrated appConfig in values.yaml, we not use it anymore, not?

**Review by web7evgheni — CHANGES_REQUESTED** [review]

**Review by web7evgheni — APPROVED** [review]

**Pull request merged**